Makefile

```diff
@@ -78,7 +85,7 @@ rpm: clean dist ##             - Creates a local RPM package, useful for develop
 ostree: qm_dropin_img_tempdir ##             - A helper for creating QM packages for ostree based distros
```

Comment:
I think that the formatting of the comments may be refined to make them more consistent and easier to read.
Adding a ## symbol before the comments to make them more visible and distinguishable from the code:
`ostree: qm_dropin_img_tempdir
	## Helper for creating QM packages for ostree based distros`